src/main/java/org/ecocean/CommonConfiguration.java

```diff
@@ -337,6 +337,58 @@ public static String getNumIaResultsUserCanInspect(String context) {
     }
   }
 
+  public static String getAdoptionCampaignUrl(String context) {
+    try {
+      return getProperty("adoptionCampaignUrl", context).trim();
+    } catch (Exception e) {
+      System.out.println(
+          "Error in getAdoptionCampaignUrl. Returning nothing as a default.");
+      return "";
+    }
+  }
```

Comment:
You could catch all the possible NPE's in this series of methods by modifying the CommonConfiguration.getProperty method. You could also change the pokemon exception to explicitly NullPointerException, I don't think there is another possibility. Totally have just used Exception out of habit myself too, nbd. Mostly trying to avoid the NPE caused by calling .trim() on a key that isn't yet known to exist, yeah?